sdb/commands/stacks.py

```diff
@@ -222,6 +223,13 @@ def call(self, objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
         #
         stack_aggr = defaultdict(list)
         for task in for_each_task(self.prog):
+            #
+            # We're not able to unwind the stack of a running task, so
+            # for any running task, we silently skip it.
+            #
+            if task_state_to_char(task) == 'R':
```

Comment:
This makes sense to me.

Unfortunately ValueError is a pretty generic exception to throw and I would love it if there was some way for us to ensure that this exception was thrown because we hit a running stack without having to do an assertion on the exception message (we could also do an assertion on the task_state but we'd hit the same issue that you outline above - what if the thread went back to a non-running state by that time?).

Let's proceed as you suggest and just catch that exception for now. We can communicate with Omar and try to figure out a clean way of dealing with this.